src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -193,10 +195,21 @@ export default class GeGiftEntryFormApp extends NavigationMixin(LightningElement
 
                 this.dataImportRecord = await upsertDataImport({ dataImport: this.dataImportRecord });
 
-                const isFailedPurchase = purchaseResponse.statusCode !== 201;
-                if (isFailedPurchase) {
+                this.isFailedPurchase = purchaseResponse.statusCode !== 201;
+                if (this.isFailedPurchase) {
                     let errors = this.getFailedPurchaseMessage(purchaseResponse);
-                    throw new Error(errors);
+                    let labelReplacements = [this.CUSTOM_LABELS.commonPaymentServices, errors];
+                    let formattedErrorResponse = format(this.CUSTOM_LABELS.gePaymentProcessError, labelReplacements);
+                    let splitErrorResponse = formattedErrorResponse.split('/newline/');
```

Comment:
This might become an issue once our custom labels are localized. We might want to consider using a hex value or something instead of 'newline' that wouldn't be translated and document it at such in the Custom Labels file to give the translation team a heads up.

Reply:
Replaced with constant 0x0A (line feed).